src/resources/images/aggregator/eventbrite-tickets.svg

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You might want to save some bytes here by compressing the SVG, a handy tool:

- https://jakearchibald.github.io/svgomg/